packages/chakra-ui-core/src/CImage/tests/CImage.test.js

```diff
@@ -0,0 +1,50 @@
+import { CImage } from '../..'
+import { render, wait, waitMs, screen } from '@/tests/test-utils'
+const LOAD_FAILURE_SRC = 'LOAD_FAILURE_SRC'
+const LOAD_SUCCESS_SRC = 'LOAD_SUCCESS_SRC'
+
+beforeAll(() => {
+  process.browser = true // Mock process.browser for CAvatar created()
+})
+
+beforeAll(() => {
+  // Mock Img
+  // eslint-disable-next-line accessor-pairs
+  Object.defineProperty(global.Image.prototype, 'src', {
+    set (src) {
+      // Mock the Image.prototype.onload/onerror implementations
+      this.onload = jest.fn().mockImplementation(fn => fn)
+      this.onerror = jest.fn().mockImplementation(err => err)
```

Comment:
Same here with mocking the `onload` and `onerror` implementations` as shared above.

Reply:
same as my previus comment.